include/insns/Zicbom.h

```diff
@@ -0,0 +1,70 @@
+//
+// _Zicbom_h_
+//
+// Copyright (C) 2017-2023 Tactical Computing Laboratories, LLC
+// All Rights Reserved
+// contact@tactcomplabs.com
+//
+// See LICENSE in the top level directory for licensing details
+//
+
+#ifndef _SST_REVCPU_ZICBOM_H_
+#define _SST_REVCPU_ZICBOM_H_
+
+#include "../RevInstHelpers.h"
+#include "../RevExt.h"
+
+#include <vector>
+#include <limits>
+
+namespace SST::RevCPU{
+class Zicbom : public RevExt{
+
+  static bool cmo(RevFeature *F, RevRegFile *R, RevMem *M, RevInst Inst) {
+    switch(Inst.imm){
+    case 0b000000000000:
```

Comment:
I know you commented the case names but maybe put them in either macros or an enum instead?

Reply:
fixed in d8b98be